packages/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.ts

```diff
@@ -701,9 +708,15 @@ export class CurrentActivityPrinter extends ActivityPrinterBase {
     return '[' + color(fullChars + partialChar) + filler + `] (${this.resourcesDone}/${this.resourcesTotal})`;
   }
 
+  private addHookFailedReason(activity: StackActivity) {
```

Comment:
Maybe just call this `failureReason` ?

Reply:
changed.        I see there is no test for this file. do u have any examples for testing?

Reply:
@rix0rrr   hey Rico, could we merge this first and add test later?